### ISSUE TITLE
Incomplete refrences between MONDO and HPO terms

### ISSUE THREAD

**Issue by fmjabato** [monarch-initiative/mondo#1990]
We are interested in using your ontology to relate diseases to Human Phenotype Ontology (HPO) terms.

We have observed that you have MONDO terms directly related to HP terms by xref tag but you use different formats like:

- (Short format) MONDO:0000088
- (Extended format) MONDO:0000063

But our main problem is that **not all MONDO terms have refrences to HP terms**. Specifically, there are only 563 MONDO terms with, at least, one HP term linked.

We have linked more HP terms using an annotation table given by HPO called "phenotype_annotation.tab" (https://hpo.jax.org/app/download/annotation) and linkage of MONDO terms with Orphanet and/or OMIM terms.

Can you include HP linkages directly into MONDO?

**Comment by nicolevasilevsky** [comment]
Hi @fmjabato - to clarify, by short and long format - do you mean xrefs with and without provenance? 

Regarding including HPO linkages directly into Mondo - let me discuss further with our team and we'll get back to you.

**Comment by nicolevasilevsky** [comment]
@fmjabato sorry for the delay on this - could you let me know about the question above when you have a chance? Thanks!

**Comment by fmjabato** [comment]
Never mind @nicolevasilevsky . Let me reformule the question.

We are working with HPO in order to study genetic diseases (coded, for example, using OMIM and ORPHANET). We are studying your MONDO ontology and we found that you directly reference MONDO items with HPO IDs (forget the short/extended matter).

If we take the direct references, we found 563 MONDO with phenotypes (HPO ids). We have used an external file (from official HPO web) which relates HPO_ids with OMIM/Orphanet ids ( [phenotype_annotation.tab](https://hpo.jax.org/app/download/annotation) ). Using your relations MONDO-OMIM/Orpha, we are able to infer new phenotype relations obtaining 4703 MONDO terms with any phenotype related.

We want to know why are not those relations into MONDO ontology? Or why we shouldn't use these HPO inferences for our studies?.

Thanks in advance

**Comment by drseb** [comment]
I think you are mixing up the concept of an "xref" and a "disease-to-clinical-feature association". These are not the same. An xref is not much - the obo guide mentions something like "This xref is of relevance to the current entity". The HPO-annotation files you are using are more, they clearly associate a disease concept (D) with individual clinical features that patients with the diagnosis D are typically showing. 
Let me know if you have further questions.

**Comment by fmjabato** [comment]
Okey, at this point I have some doubts. I understand that the HPO file relates phenotypes with diseases. At this point, I'm interested on use your MONDo hierarchy to work with MONDO items.

Specifically, I want to assign phenotype profiles (HPO IDs) to MONDO terms in order to apply some semantic and other mathematical studies.

And I do the following:

1. I use your mondo.obo file and use **xref** tag to obtain MONDO-{HPO/OMIM/Orphanet} tuples. At this point, I find 563 MONDO terms with any (>=1) phenotypes (HPO id) linked. Note: there are several types of info relate to MONDO items using xref tags but we are parsing to take only these three.
2. I use HPO annotation file - with HPO-{OMIM/Orpha} tuples - and already loaded MONDO-{OMIM/Orpha} tuples to infer new MONDO-HPO relations. At this point, I increase the MONDo with any size phenome to 4703.

Both of you (Monarch and HPO) perform statistical inferences based on graphs in order to obtain referenced files (mondo.obo and phenotype_annotation.tab).

I want to know if **xref referenced on your MONDO OBO format file are significant?** or how are you inferring it to your MONDO terms? It it are relevant, I want to know why there are not being referenced new relations obtained after 2) process.

**Comment by matentzn** [comment]
Is your question @fmjabato whether Mondo contains all the xrefs where a Mondo ID equals an HP ID? In our conceptual framework, which may be confusing to some, diseases (Mondo) and phenotypes (HPO) are never "the same" even if the labels sound similar. Are you trying to merge HPO and Mondo terms like that? And is your questions wether we have _all_ of them in Mondo? Then the answer is no.. Its not complete.

**Comment by matentzn** [comment]
As this ticket is very old, I will close it now, but @fmjabato, please reopen this ticket if this is still and open issue for you - happy to help you sort it out :) Sorry for the delay!
